modules/e2ee/Context.js

```diff
@@ -39,7 +39,7 @@ export class Context {
     /**
      * @param {string} id - local muc resourcepart
      */
-    constructor(id) {
+    constructor(externallyManaged = false) {
```

Comment:
We could create the context with shared keys as follows: `new Context({ shareKey: true });

Then, internally, we wouldn't derive any keys or ratchet. We just use what we are given in setKey.